src/conn/conn_oligarch.c

```diff
@@ -1019,6 +1111,9 @@ __wti_disagg_destroy_conn_config(WT_SESSION_IMPL *session)
         disagg->bstorage_meta = NULL;
     }
 
+    __wt_free(session, disagg->page_log);
```

Comment:
Why we need to put this to the end of the function?

Reply:
Just in case we would use these parameters in a destructor in the future, but it's not necessary. All these three parameters are strings, so we may as well leave them until the end just in case.